torch/ao/quantization/backend_config/_common_operator_config_utils.py

```diff
@@ -353,6 +353,7 @@ def _get_default_op_configs(dtype_configs: List[DTypeConfig]) -> List[BackendPat
         torch.nn.LayerNorm,
         torch.nn.Dropout,
         torch.nn.PReLU,
+        torch.nn.ChannelShuffle,
```

Comment:
should add this to _get_share_qprams_op_backend_config I think

Reply:
I have added channel shuffle to copy node list. Is this still needed?